ngraph/core/src/op/non_zero.cpp

```diff
@@ -118,10 +118,13 @@ namespace nonzero
 #define TYPE_OUT_CASE(a, ...)                                                                      \
     case element::Type_t::a:                                                                       \
     {                                                                                              \
-        NGRAPH_OP_SCOPE(OV_CC_CAT3(evaluate_nonzero_out, _, a),                                    \
-                        rc = evaluate_nonzero_execute<INPUT_ET, element::Type_t::a>(__VA_ARGS__)); \
+        NGRAPH_OP_SCOPE(OV_CC_CAT3(evaluate_nonzero_out, _, a))                                    \
+        {                                                                                          \
+            rc = evaluate_nonzero_execute<INPUT_ET, element::Type_t::a>(__VA_ARGS__);              \
+        }                                                                                          \
     }                                                                                              \
-    break;
+    break
```

Comment:
`;` ?

Reply:
It is a redundant `;` because in switch/cases we always use:
```
TYPE_OUT_CASE(a, ...);
```

Reply:
Align between files: see ngraph/core/src/op/one_hot.cpp

Reply:
Fixed